examples/pylab_examples/print_stdout.py

```diff
@@ -1,13 +1,9 @@
-#!/usr/bin/env python
-# -*- noplot -*-
-# print png to standard out
```

Comment:
Sorry I actually meant the last two lines but the plot directive should stay too